.azure-pipelines/sync-fork.yml

```diff
@@ -0,0 +1,54 @@
+parameters:
+  - name: fork_repo_url
+    displayName: fork_repo_url
+    default: https://github.com/microsoft/openjdk-jdk
+  - name: upstream_repo_url
+    displayName: upstream_repo_url
+    default: https://github.com/openjdk/jdk
+  - name: fork_branch
+    default: main
+    displayName: fork_branch
+  - name: upstream_branch
+    default: master 
+    displayName: upstream_branch
+
+trigger: none
+
+schedules:
+  # At 5am UTC (10pm PST) daily
+  - cron: "0 5 * * *"
+    displayName: Sync
+    always: false # only run if there has been code changes to the jdk codebase
+    branches:
+      include:
+        - "*"
```

Comment:
```suggestion
        - "master"
        - "main"
```

Reply:
I think we want to only sync when upstream/master has changed (so maybe we don't need master in there?)